applications/NXmx.nxdl.xml

```diff
@@ -267,104 +371,115 @@
 									<item value="translation" />
 								</enumeration>
 							</attribute>
-							<attribute name="vector">
+							<attribute name="vector" type="NX_NUMBER">
 							</attribute>
-							<attribute name="offset">
+							<attribute name="offset" type="NX_NUMBER">
```

Comment:
I am confused here, perhaps I am missing something. I understand that the "module_offset" field has the direction and magnitude given by the "vector" and "offset" attribute, respectively. However, shouldn't a field have a value associated with it? If all the relevant info is given by attributes, then what is left to write in the field value?
The same issue is repeated a few times throughout the NXdetector_module base class, so it isn't just an NXmx issue.

Reply:
It's specified [in](https://manual.nexusformat.org/design.html#coordinate-transformation-field-and-attributes) for all NXtransformations fields.

Reply:
I don't know.  My guess would be that, as with axis setting, we could give a value (or values) that would then have the effect of giving a displacement from the offset in the
direction of the vector, but if there actually is a dynamic positioner it would be better
to explicitly describe the relevant positioner axis and its settings.  For the moment
I will go forward with the current pull request and suggest we discuss this point further in the future, perhaps at NIAC.